test/over_react/component_declaration/builder_integration_tests/component2/constant_required_accessor_integration_test.dart

```diff
@@ -196,38 +186,42 @@ void main() {
           attachedToDocument: true,
         );
 
-        expect(consoleErrors, isEmpty, reason: 'should not have outputted a warning but found: $consoleErrors');
-
-        jacket.rerender((ComponentTest()
-          ..required = true
-          ..nullable = null
-          ..requiredAndLengthLimited = [1,2]
-        )());
+        expect(() {
+          jacket.rerender((ComponentTest()
+            ..required = true
+            ..nullable = true
+            ..requiredAndLengthLimited = [1,2]
+          )());
+        }, logsNoPropTypeWarnings);
```

Comment:
Same question here - can the `logsNoPropTypeWarnings` assertion not be made using the return value of `mount` above?